client/my-sites/plans/jetpack-plans/storage-tier-upgrade/hooks/use-get-product-card-data.ts

```diff
@@ -0,0 +1,205 @@
+import {
+	useJetpackStorageAmountTextByProductSlug,
+	getJetpackProductDisclaimer,
+	ProductSlug,
+	PlanSlug,
+	PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_10GB_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_100GB_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_1TB_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_3TB_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_5TB_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_T1_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_T1_YEARLY,
+	PRODUCT_JETPACK_BACKUP_T2_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_T2_YEARLY,
+	PLAN_JETPACK_SECURITY_T1_MONTHLY,
+	PLAN_JETPACK_SECURITY_T1_YEARLY,
+	PLAN_JETPACK_SECURITY_T2_MONTHLY,
+	PLAN_JETPACK_SECURITY_T2_YEARLY,
+} from '@automattic/calypso-products';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import { useCallback, useMemo } from 'react';
+import slugToSelectorProduct from 'calypso/my-sites/plans/jetpack-plans/slug-to-selector-product';
+import type { SelectorProduct } from 'calypso/my-sites/plans/jetpack-plans/types';
+
+type StorageUpgradeGetter = ( slug: string, isPurchased: boolean ) => SelectorProduct;
+
+const MONTHLY_BACKUPS_PRODUCTS = [
+	PRODUCT_JETPACK_BACKUP_T1_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_T1_YEARLY,
+	PLAN_JETPACK_SECURITY_T1_MONTHLY,
+	PLAN_JETPACK_SECURITY_T1_YEARLY,
+	PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_10GB_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_100GB_MONTHLY,
+];
+
+const YEARLY_BACKUPS_PRODUCTS = [
+	PRODUCT_JETPACK_BACKUP_T2_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_T2_YEARLY,
+	PLAN_JETPACK_SECURITY_T2_MONTHLY,
+	PLAN_JETPACK_SECURITY_T2_YEARLY,
+	PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_1TB_MONTHLY,
```

Comment:
These 3 should be in the other array, no?

Reply:
These are products that have a one-year activity log and restore period. It's confusing indeed, I'll update the name 👍